rules/sonic-py-common.mk

```diff
@@ -4,6 +4,8 @@ ifeq ($(ENABLE_PY2_MODULES), y)
     SONIC_PY_COMMON_PY2 = sonic_py_common-1.0-py2-none-any.whl
     $(SONIC_PY_COMMON_PY2)_SRC_PATH = $(SRC_PATH)/sonic-py-common
     $(SONIC_PY_COMMON_PY2)_DEPENDS += $(SWSSSDK_PY2)
+    $(SONIC_PY_COMMON_PY2)_DEBS_DEPENDS = $(LIBSWSSCOMMON) \
```

Comment:
>LIBSWSSCOMMON [](http://example.com/codeflow?start=44&length=13)

Does it depend on LIBSWSSCOMMON directly?

Reply:
Fixed, remove indirect dependency to LIBSWSSCOMMON.